python/test/unit/language/test_core.py

```diff
@@ -7332,3 +7332,13 @@ def dtype_tensor_kernel(dtype: tl.constexpr):
         tensor = tl.zeros((1, ), dtype)
 
     dtype_tensor_kernel[(1, )](dtype)
+
+
+@pytest.mark.interpreter
+def test_float_tuple():
```

Comment:
I think we test float infs here instead of tuples.
```suggestion
def test_float_inf():
```